README.md

```diff
@@ -1,2 +1,123 @@
 # shop-theme
-Magento2 theme for RWAHS shop
+
+Magento2 theme for RWAHS shop, based on the pre-packaged Luma theme.
+
+The following guide uses the following directory layout:
+
+```
++ /
+|
++-+ srv/
+  |
+  +-+ deploy/
+  | |
+  | +-- shop-theme/ (this repo is cloned here)
+  | |
+  | +-- shop-migration/ (the shop-migration repo is cloned here)
+  |
+  +-+ sites
+    |
+    +-- <environment>-shop.histwest.org.au/ (magento install path)
+```
+
+Where `<environment>` is the name of the environment being installed, e.g. `staging` or `production`.  In the commands 
+below, the `<environment>` used is `documentation`.
+
+The directories indicated above are used in commands below; if you want a different file system layout, you must alter 
+the commands accordingly.
+
+The database host is assumed to be `<environment>-shop-db.histwest.org.au`, the database name is `shop`, the database 
+user is `magento2` and the password is `password`.  Alter these values as appropriate.
+
+## Reference
+
+These instructions follow the basic principles outlined in the Magento 2.0 documentation:
+
+* http://devdocs.magento.com/guides/v2.0/install-gde/bk-install-guide.html
+
+Note that no consideration is given in the instructions below to file system ownership and permission, however it is 
+assumed that the Magento guidelines are adhered to:
+
+* http://devdocs.magento.com/guides/v2.0/install-gde/prereq/integrator_install.html#mage-owner-about-group
+
+Finally, server software is required:
+
+* A web server; `nginx` is recommended, in combination with `php-fpm`.  Setup of the web server or CGI backend is 
+outside of the scope of this document.
+* A database server; `mysql` or a variant is required by Magento.  These instructions assume a database has been 
+created and permissions assigned, as per standard procedure.
+
+## Steps
+
+### 1. Clone repositories
+
+```
+mkdir -p /srv/deploy
+cd /srv/deploy
+git clone https://github.com/rwahs/shop-theme.git
+git clone https://github.com/rwahs/shop-migration.git
+```
+
+### 2. Install Magento code
+
+* Version: 2.0.x
+* Method: composer
+
```

Comment:
You also need an access key as defined at http://devdocs.magento.com/guides/v2.0/install-gde/prereq/connect-auth.html